web/public/js/init-master.js

```diff
@@ -859,6 +859,18 @@
     .run(['$rootScope', '$filter', '$state', 'embedder', '$cookieStore', '$document', 'verifyProfileComplete', 'alertService', '$translate', '$location',
      function($rootScope, $filter, $state, embedder, $cookieStore, $document, verifyProfileComplete, alertService, $translate, $location){
 
+      // Override $translate.instant so it falls back to en_US, then the original key when no result
+      var originalTranslateInstant = $translate.instant;
+      $translate.instant = function (key) {
+        var translation = originalTranslateInstant.apply($translate, arguments);
+        if (!translation) {
+          var args = Array.prototype.slice.apply(arguments); // Needed so we can modify the arguments
+          args[3] = 'en_US'; // The forceLanguage argument
+          translation = originalTranslateInstant.apply($translate, args);
```

Comment:
Ok, that was because I forgot to commit the file that returns the dictionaries based on the URI param.

On your comment then, yeah, if the key doesn't exist, then translation will become the key here, but if the key does exist, but for some reason the value is an empty string in the en_US messages.po, then we want to default to the key, hence the `|| key`